constants/map_object_constants.asm

```diff
@@ -104,17 +104,24 @@ DEF MAPOBJECT_Y_COORD          rb ; 2
 DEF MAPOBJECT_X_COORD          rb ; 3
 DEF MAPOBJECT_MOVEMENT         rb ; 4
 DEF MAPOBJECT_RADIUS           rb ; 5
-DEF MAPOBJECT_HOUR             rb ; 6
+DEF MAPOBJECT_HOUR_1           rb ; 6
+DEF MAPOBJECT_HOUR_2           rb ; 7
+rsset MAPOBJECT_HOUR_2            ; HOUR_2 and TIMEOFDAY are the same byte
 DEF MAPOBJECT_TIMEOFDAY        rb ; 7
-DEF MAPOBJECT_COLOR            rb ; 8
-DEF MAPOBJECT_RANGE            rb ; 9
+DEF MAPOBJECT_PAL_OVERRIDE     rb ; 8
```

Comment:
```suggestion
DEF MAPOBJECT_PALETTE          rb ; 8
```

Reply:
I know there's a "`.skip_color_override`" label in the source, but IMO it's simpler to call this the object's palette (with 0 meaning it's a default for the sprite).